### PR TITLE
modify readMe bug

### PR DESCRIPTION
**Pull request by Kelang-Tian**
**First pull request test.**

>docker build -t sqlflow:dev . -> docker build -t sqlflow:dev -f docker/dev/Dockerfile .
[https://github.com/sql-machine-learning/sqlflow/blob/develop/docker/dev/README.md](url)

The documentation is imperfect. 
You need to tell the user to execute the command in the root directory and specify the location of the dockerfile.

### CODE REVIEW

File: docker/dev/README.md [hunk 1]
```diff
@@ -7,10 +7,10 @@ code of SQLFlow.
 
 ## Build the DevBox Image
 
-In this directory, run the following command.
+In the root directory of this project, run the following command.
```
Comment (Yancey0623) [inline docker/dev/README.md]: Please trim the last space character.

**Review by Yancey0623 — COMMENTED** [review]
LGTM, Thanks to polishing this tutorial, I left a tiny comment, please update as it and then we can merge this PR.

File: docker/dev/README.md (hunk 1 above)
Comment (Kelang-Tian) [reply docker/dev/README.md]: Ok

**Review by Yancey0623 — APPROVED** [review]
LGTM

**Pull request merged**